controller/cache/cache.go

```diff
@@ -38,7 +38,7 @@ type LiveStateCache interface {
 	Invalidate()
 }
 
-type AppUpdatedHandler = func(appName string, fullRefresh bool, ref v1.ObjectReference)
+type AppUpdatedHandler = func(appName string, isManaged bool, ref v1.ObjectReference)
```

Comment:
what does `isManaged` mean?

Reply:
IsManaged indicates if it is managed resource (e.g. specified in git or not). Renamed variable accordingly.